domain/entities/accounts_test.go

```diff
@@ -28,15 +28,15 @@ func TestNewAccount(t *testing.T) {
 
 		HashedPIN, _ := HashPIN(pin)
```

Comment:
aparentemente faltou esse daqui, variáveis locais devem começar com letra minúscula:
```suggestion
		hashedPIN, _ := HashPIN(pin)
```

Reply:
subi, Pedro! obrigada